discontinuous-range/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for discontinuous-range 1.0
+// Project: https://www.npmjs.com/package/discontinuous-range
+// Definitions by: Victor Zhou <https://github.com/OiCMudkips>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Represents a discontinuous range of numbers.
+ */
+export declare class DiscontinuousRange {
```

Comment:
The library uses `module.exports = DiscontinuousRange;`, which would correspond to `export = DiscontinuousRange; declare class DiscontinuousRange { ... }`. Please verify that these definitions work for you at runtime.